corehq/apps/domain/project_access/models.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from django.db import models
+
+ENTRY_RECORD_FREQUENCY = datetime.timedelta(hours=6)
+
+
+class SuperuserProjectEntryRecord(models.Model):
+    username = models.EmailField()
```

Comment:
I think this should have an index on it. It may be useful to look at all of someone's actions if their account is compromised. or maybe even a proactive email that says "this is what you did today"